src/com/griddynamics/cinema/HallRow.java

```diff
@@ -0,0 +1,36 @@
+package com.griddynamics.cinema;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class HallRow {
+    
+    private final List<Seat> seats;
+    int rowPosition;
```

Comment:
Why it’s package-private?

Reply:
Didn't notice this mistake. It will be changed to private